pkg/shp/cmd/buildrun/create.go

```diff
@@ -73,10 +75,25 @@ func (c *CreateCommand) Run(params *params.Params, ioStreams *genericclioptions.
 	br := &buildv1alpha1.BuildRun{Spec: *c.buildRunSpec}
 	flags.SanitizeBuildRunSpec(&br.Spec)
 
+	var build = buildv1alpha1.Build{}
+	if err := resource.GetBuildResource(params).Get(c.cmd.Context(), c.buildRunSpec.BuildRef.Name, &build); err != nil {
```

Comment:
Looking forward to the switch from dynamic client to versioned one. I did that in our `build-load` tool, too. Actually, I started to work on that, because I wanted that switch before doing the local source code. However, it turned out to be more work than I anticipated and I then switched back and decided to first do this one here..